modules/dynamic-operator/plugin/postgresql/main.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
```

Comment:
- Let's move this outside of plugin directory to something like example-plugin
- ok